doc-content/jbpm-docs/src/main/asciidoc/DesignBuildCases/case-management-case-expression-proc.adoc

```diff
@@ -23,7 +23,7 @@ The following procedures uses the *IT_Orders* sample project to demonstrate how
 .Setting the UPPER function for the customCaseIdPrefix metadata attribute
 image::cases/expressions.png[]
 
-In this example, `type` is a variable set in the *Case File Variables* field, which during runtime a user may define to it the value `type1`. `UPPER` is a pre-built function to uppercase a variable, and `IT-` is a static prefix. The results are dynamic case IDs such as `IT-TYPE1-0000000001`, `IT-TYPE1-0000000002`, and `IT-TYPE1-0000000003`.
+In this example, `type` is a variable set in the *Case File Variables* field, which during runtime a user can define to it the value `type1`. `UPPER` is a pre-built function to uppercase a variable, and `IT-` is a static prefix. The results are dynamic case IDs such as `IT-TYPE1-0000000001`, `IT-TYPE1-0000000002`, and `IT-TYPE1-0000000003`.
```

Comment:
"which during runtime a user can define to it the value `type1`" makes no sense to me. Maybe...
"In this example, `type` is a variable set in the *Case File Variables* field. At run time, the user can assign the value `type1` to this variable."